config.go

```diff
@@ -166,7 +189,7 @@ func writeConfig(f Function) (err error) {
 }
 
 // validateVolumes checks that input Volumes are correct and contain all necessary fields.
-// Returns array of error messages, empty if none
+// Returns array of error messages, empty if none errors are found
```

Comment:
```suggestion
// Returns array of error messages, empty if no errors are found
```
Such a tiny nit here and below...

Reply:
Thx!